iosMath/render/MTMathListDisplay.m

```diff
@@ -200,7 +208,9 @@ - (void)setTextColor:(MTColor *)textColor
     // Set the color on all subdisplays
     [super setTextColor:textColor];
     for (MTDisplay* displayAtom in self.subDisplays) {
-        displayAtom.textColor = textColor;
+        if (displayAtom == nil) {
```

Comment:
This check doesn't seem to make sense to me. Why displayAtom == nil?
Also, objective-C doesn't really need a != nil check either.